networkx/drawing/layout.py

```diff
@@ -88,6 +88,11 @@ def random_layout(G, center=None, dim=2, seed=None):
         if None, the random number generator is the RandomState instance used
         by numpy.random.
 
+    store_pos_as : str, default None
+        If non-None, the position of each node will be stored on the graph as
+        an attribute named `attribute` which can be accessed with
```

Comment:
This is just a suggestion for wording (remove the ```named `attribute` ``` text which implies the name will literally be "attribute"). But it will need to be changed for all the layouts, so I'm not making it a github suggestion.
```
-        an attribute named `attribute` which can be accessed with
+        an attribute with this string as name so it can be accessed with
```

Reply:
This is definitely an improvement to the wording of the docstring.